schemas/configuration/generate.json

```diff
@@ -0,0 +1,223 @@
+          }
+        },
+        "then": {
+          "properties": {
+            "model_csv": {
+              "type": "string",
+              "format": "file",
+              "description": "Provide the model CSV here. Example template can be seen on [docs.meshery.io](https://docs.meshery.io)",
+              "x-rjsf-grid-area": "6"
+            },
+            "component_csv": {
+              "type": "string",
+              "format": "file",
+              "description": "Browse the filter file from your file system",
+              "x-rjsf-grid-area": "6"
+            },
+            "relationship": {
+              "type": "string",
+              "format": "file",
+              "description": "Browse the filter file from your file system",
+              "x-rjsf-grid-area": "6"
+            }
+          },
+          "required": ["model_csv", "component_csv"]
+        }
+      },
+      {
+        "if": {
+          "properties": {
+            "uploadType": {
+              "const": "URL Import"
+            }
+          }
+        },
+        "then": {
+          "properties": {
+            "url": {
+              "type": "string",
+              "format": "uri",
+              "title": "URL",
+              "description": "Provide the URL of the design file you want to import.",
+                "x-rjsf-grid-area": "12"
+            },
+            "model": {
+              "type": "object",
+              "description": "Provide the details of the model you are uploading incase you are trying to generate a new model",
+              "properties": {
+                "modelDisplayName": {
+                  "description": "The model name that would be showed on kanvas and everywhere else mostly of type 'Model Name'",
+                  "type": "string"
+                },
+                "registrant": {
+                  "title": "Source",
+                  "type": "string",
+                  "enum": ["github", "artifacthub"]
+                },
+                "model": {
+                  "description": "This name is of the type 'model-name'",
+                  "type": "string"
+                },
+                "category": {
+                  "description": "The category of the model. The model will be located under the specific category on kanvas.",
+                  "type": "string",
+                  "enum": [
+                    "Analytics",
+                    "App Definition and Development",
+                    "Cloud Native Network",
+                    "Cloud Native Storage",
+                    "Database",
+                    "Machine Learning",
+                    "Observability and Analysis",
+                    "Orchestration & Management",
+                    "Platform",
+                    "Provisioning",
+                    "Runtime",
+                    "Security & Compliance",
+                    "Serverless",
+                    "Tools",
+                    "Uncategorized"
```

Comment:
These should be fetched dynamically, look for the IMport design model (source type field) for reference on dynamically fetching.

Reply:
@MUzairS15 We used enum here to :thinking: 
https://github.com/meshery/meshkit/blob/fe3024a1caa43e6e9b281c326dac83bb93f93df7/schemas/configuration/designImport.json#L13

Reply:
Then it must be some other schema, check catalog publish

Reply:
I'll make them dynamically after today's event.